deployment/tasks/remove_environment.cap

```diff
@@ -0,0 +1,8 @@
+namespace :symfony do
+    desc "remove development environment"
+    task :remove_dev_environment do
+        on roles(:env) do
+            execute :php, "rm -f #{fetch(:deploy_to)}/current/web/app_*"
```

Comment:
I'll add the suffix '.php' to the file pattern is possible to avoid removing other files starting with 'app_'